MAUI/Expander/Getting-Started.md

```diff
@@ -0,0 +1,400 @@
+---
+layout: post
+title: Getting Started with .NET MAUI Expander control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI Expander (SfExpander) control, its elements and more.
```

Comment:
Description: Learn about getting started with Syncfusion .NET MAUI Expander (SfExpander) control, its elements, and more.